apps/desktop/desktop/src/main/resources/css/controls.css

```
@@ -128,6 +129,52 @@
 }
 
 
+.buy-button {
+    -fx-background-color: -bisq2-green-dim-40;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)